waspc/waspc.cabal

```diff
@@ -6,7 +6,7 @@ cabal-version: 2.4
 --    Consider using hpack, or maybe even hpack-dhall.
 
 name:           waspc
-version:        0.14.0
+version:        0.14.1
```

Comment:
Let's revert this change as well since we'll be releasing 0.14.0 in mid-July and not 0.14.1 (I'm confident we'll merge this until then)